README.md

```diff
@@ -91,22 +91,32 @@ To set up the app for local development you need to follow these steps:
     ```sh
     cp instance/config.py.sample.dev instance/config.py
     ```
+2. Start rds-client-api and mysql 
+    ```docker-compose up -d```
 
-2.  Fill in the missing configuration values as per the **Configuration
+
+3. Make sure the database is up to date.
+   ```
```

Comment:
It would be worth mentioning here that a relevant venv should be created and the pip packages installed